### DISCUSSION TITLE
投资炼金术 · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#118] [Announcements]
# 投资炼金术 · 构建我的被动收入

投资炼金术系统支持多种投资交易策略，可评估用户的投资交易水平，自动计算用户投资组合的净值，从而提升用户的投资交易水平。

https://www.bmpi.dev/money/invest-alchemy/

**Comment by GitHubxhwang** [comment]
最近也想做一些投资策略的模拟，希望可以有支持用户自行整理策略，进行数据回测。邮件提醒方面也可以添加用户自己设定策略进行跟踪提醒。

**Comment by madawei2699** [comment]
跟市面上的量化软件有何区别？

**Reply by madawei2699** [reply]
投资炼金术更偏向于投资组合的整体风险与收益评价，很多量化软件更专注与策略开发回测，并不适合普通投资者，这个系统能满足我对投资的需求：简单的策略、长期投资。那么我需要知道我的投资组合有何风险，交易表现如何，这是这个系统可以帮助我发现自身问题的。量化系统一般不关注投资组合整体的风险与收益。

**Comment by madawei2699** [comment]
为什么很多投资策略并不适合做股票交易？

**Reply by madawei2699** [reply]
投资炼金术的投资交易策略主要交易指数基金而非股票，是因为相比指数，股票存在停牌和流动性的风险，这两个都能造成无法交易的风险，模拟交易一般很难考虑这两者的情况，所以模拟交易好的实际效果可能很差，但指数基金没有停牌的风险，大的指数基金也没有流动性的风险，所以模拟交易和实际情况差距不大。

**Comment by madawei2699** [comment]
投资炼金术适合哪些用户？

**Reply by madawei2699** [reply]
为了满足上班族或业余投资者简单长期的投资需求，投资炼金术这个辅助用户投资交易的系统，它可以从投资组合整体的角度评价交易策略的风险与收益，而不像大多量化投资软件，解决了交易策略在模拟回测与投资组合实践中差距过大的问题。

**Comment by madawei2699** [comment]
V2EX讨论贴：https://www.v2ex.com/t/879524
Twitter讨论贴：https://twitter.com/madawei2699/status/1568015722936430592?s=21&t=htqC6r3SsX9_lwq3ogKQwg

**Reply by madawei2699** [reply]
Twitter社区中有价值的讨论：

> madawei: 做了一个投资交易助手，支持不同投资交易策略，支持机器人自动根据资金策略与交易策略创建交易组合，支持交易组合的净值计算与风险指标量化测评，能评价个人的交易水平。目前只支持A股。

> gkiwi: 佣金好像没算在内吧（只翻了一点代码~~）另外用 backtrader 库应该可以省掉很多不必要的代码，可以试一试

> madawei: 因为都是etf场内基金，佣金其实很低，所以没计算，但数据库表中保留了这个字段。backtrader我试试，现在策略信号实现都是直接手工处理的，如果用回测框架去搞会不会更优雅些🐶

> gkiwi: 我自己没交易过场内etf，不过自己做其他的经验就是万二佣金频次上去了你就会发现根本不是那么回事，收益会被吃掉很多。另外回测和真实的效果差很多，一言难尽，加油共勉。。backtrader 值得拥有。

> madawei: 我的策略都不是高频，几年的交易次数也就几十到一二百次，所以佣金真不是个问题，这和策略有关系。回测和真实差距大是因为策略针对的如果是股票，那滑点、停盘与流动性都会造成很大的偏差，但场内指数基金一不会停牌，二有很好的流动性，所以模拟回测和真实之间差距很小，所以策略的标的不同会有影响

> gkiwi: 不知道你的佣金是多少，百次交易也不少吧，如果真有万二倒是真无所谓。回测与真实差距大，我到不认为是你说的那几个点，虽然这些因素存在。在我看来更多的是我们设置的条件是为了追求高收益率，导致的过拟合，从而产生的偏差。指数对政策导向依然敏感，而这是无法通过算法预测的。

> madawei: 佣金真不多，指数基金一般一次交易就几块钱，我自己统计过，所以觉得这个因素无所谓。倒是融资成本是需考虑的。过拟合那是回测调优的问题，这个在双均线这种策略的影响也不大，不过在回测其他策略时会有这个风险，但这类风险难避免，还有如系统失灵或死亡的风险等，这些都是交易里不可知的部分和魅力。

> gkiwi: 那确实佣金可以不考虑了。过拟合问题，好奇你是如何调优解决的？单以指数基金来看，如果策略在创业板有效，是拿着这个策略在其他指数基金尝试么？

> madawei: 没有调优，就是那一个经验值去试，以双均线为例，我可以按11/22，也可以不断调整这个值去找最优的过拟合参数，但随便拿一个参数来说都可以，因为它是一个趋势跟随的策略，优缺点都很明确，适合趋势行情，在震荡行情就是成本，小亏博大赢，注定它是一个胜率不高但盈亏比很高的策略，所以无需最优就可以

> madawei: 如果有调优的话，就是让参数或策略更粗糙一些，策略越简单、组合交易标的越多，这个策略的稳定性更强，虽然收益率可能要低一些，但只要比同期全球主流指数的排名中等偏上就可以了（因为我们无法预知某个指数收益率高低），这也是这个系统关注的核心：关注投资组合整体而非某个策略。

**Reply by madawei2699** [reply]
V2EX社区中有价值的讨论：

> @grindsgears 讲真，你这种折腾未必能跑过大盘，，因为资金管理很难做， 需要预留资金为未来可能信号，而且在 A 股，大部分时间都是震，这种持有的周期太长了，除非有 tail ，不然很容易震的不要不要，就算一年抓几次 tail ，因为这种资金管理，也重不了仓。我一般 etf 只持有 2 ～ 3 天，时不时弄点 put 防防身，跑赢大盘稳稳的，lol
> 大部分人都不要折腾这种东西，没有什么意义， 如果实在要做，买点 510050 ，510300 ，510500 ，512100 ，159915 最多加个 510900 ， 基本覆盖整个市场，资金全部分配， 不要弄什么择时， 有闲钱就全部平均投进去，可以比绝大部份择时系统稳。

> @bmpidev2019 @grindsgears 你先看看我这些组合再说吧。。也许你的核心观点是没问题的，一般人折腾设计很难跑赢大盘的，但是这也不是意味着你只要拿着几个指数啥都不管就行了，我期望的是提升自己的交易水平和组合管理能力，我把它当作一个技能来提升，收益是结果而已。光拿指数也得看在山顶还是山谷，无脑拿指数或公募基金也会被套牢。

> @grindsgears  @bmpidev2019 我就是看了你的组合才给你的建议的，因为你在底层设计的时候就有问题。为了等待信号而预留资金， 又不能上杠杆或者期权，收益率实在有限。 你之前的那些测试，根本就没有考虑流动性和你下单对市场流动性的冲击， 一些小 etf ，几万的买单都能影响市场， 更不要说你进出的时候根本实现不了测试的 limited 价格。 再有，双均线这种东西，lag 太高了， 冲进去已经有人在做 mean rev 了， 除非 tail events 你不计成本冲进去。

> @bmpidev2019 @[grindsgears](https://v2ex.com/member/grindsgears) 这种操作很好模拟，但最直接的就是你在牛市顶峰去满仓买入然后一直持有，到现在还是亏的。至于你说资金效率利用不高，除非整个市场行情都是向下或震荡的，但 ETF 可以投资各个主流市场，不太可能出现这种极致的行情，但我把资金利用率也作为一种成本去看待，还有你说拿 put 去对冲，这种玩法已经不适合普通人了，没必要把这些工具也提出来。
> @[grindsgears](https://v2ex.com/member/grindsgears) 又认真思考了下你的建议，我感觉 get 到你的观点了，我之前总结过一个交易的 ppt ，你可以看看，你说的这些问题的确都存在，但不同策略都是在简单性、风险收益比、资金利用率、对使用者的心理需求方面做权衡，要追求阿尔法很难啊，起码我做不到，我选择了成本最低最简单收益率还可以接受的策略，至于有没有更好更强的，那肯定有，但这种策略对使用者有很高的要求，甚至我也尝试过这类复杂的策略，但由于对时间和心力的消耗，让我放弃了。
> 我的 ppt 在此，是一个总结类的，可能存在很多问题，欢迎指正：https://www.bmpi.dev/money/road_to_trading/

> @grindsgears @[bmpidev2019](https://v2ex.com/member/bmpidev2019) 讲真，我觉得你现在是一个非常麻烦的状态。 你了解的东西确实很多。那个 ppt 可以反映出来，你估计也学了不少年。但是，我个人觉得你还是没有找到你核心的 edge ，所以，你的 ppt 里面有很多自相矛盾的东西，比如->我们用“技术分析”建立自己的交易系统，可以你又大谈市场企业估值。你这种状态的特点就是很多人都强调纪律，交易心理之类的。 因为他们不知道自己究竟拥有什么样的 edge ？ 核心问题就是“你赚了谁的钱？”。“为什么你能赚到他们的钱？” 。 我遇到很多有 edge 的人，他们的 edge 都不一样，没有人说什么心理纪律，更不会像你 ppt 里面那种，太多摘抄。edge 就是。你和别人打架，“我比对方重 50kg ，练了 10 年 mma ，对方是个小混混” / “对方只有一个人，我有 10 个人”这种。 你 ppt 里面，很多东西在我看来，完全不合理或者我根本不知道为什么。 比如“止损过多导致资金回撤”-> 如果是 trend follwing ，那必然止损会很多很多，因为你要抓住的就是那一次 trend 。 ”识别震荡和趋势“--->如果能识别出来，往往都是事后才能识别，lol ， “技术指标的背离，表示逆转趋势，而且成功率高“ ---> 成功率 99%每次赚一块，1%一次亏 10000. 包括你的测试，2013 ～ 2015 测试 510500-->你可以试下 2008 那年 lol 。 你最后写的那个 10 条的交易系统也是这些问题。包括技术指标，能不能回答这个问题-> “为什么 MACD 金叉就会涨？”。大部分都是从结果找原因。
总之，给你的建议就是，你的这些知识，正是是阻止你交易成功的障碍， 因为你还是没有接触过核心问题---“你赚了谁的钱？”。“为什么你能赚到他们的钱？” 而不是最后那个交易纪律，好像那个什么利弗莫尔是不是也爆仓？。这是一个很专业的领域，你能接触到的这些知识，几乎都没有什么用，不然就没有人跑不赢大盘了，反而是你不知道的才是你需要了解的，比如市场流动性，还有些更底层的东西。 最好的学习是看每个交易所的网站，尤其是产品规则和交易规则，估计把交易所网站里面内容都了解的人很少很少。
如果你还是很明白， 那么更简单的办法， 看看你身边那些人赚钱，如果你身边大部分人都跑不过大盘，那么就简单了，只要不做他们做的事情，不就稳了。他们都在学技术，学基本面，那就一定不要做，他们都在弄系统量化，那就不要弄，lol
希望能帮到你，祝你好运。
> @[bmpidev2019](https://v2ex.com/member/bmpidev2019) 还有一点，运气也是一个很重要的东西，很容易迷惑你，，我给你看我发的不是我的收益率而是我交易的频率 。 如果一年年你交易 5 ～ 10 次，收益率也很高，搞不好是运气， 就像很多人在 2015 ，2020 那种。 我今年已经清仓了 130 次，平均持仓不到 3 天，这种运气的几率就比较低了。

> @nguoidiqua @[grindsgears](https://v2ex.com/member/grindsgears) 时间损耗的问题其实楼主也是有回测过的，某些标的确实一直拿着比双均线收益要更高，但有些双均线效果更好。无论长线短线什么策略，总是会有很多垃圾时间，等待机会本来就是投资的一部分。
> 对于多数懒得下功夫学习投资的人来说做好均衡配置定期再平衡确实是更好的选择，但平均而言每十个人人里面就有一个能超过市场，别人或许就有那水平，没有太大的必要去劝阻别人追求更高收益的尝试，这又不是玩什么高杠杆的东西。
> @[bmpidev2019](https://v2ex.com/member/bmpidev2019)
> 对于楼主的策略组合，个人感觉选择的标的太多了，一方面有很多重合指数没有分散作用反而增加眼睛负担，另一方面有些指数不适合这种策略拉低了收益。如果集中一点，选择十种左右相关度较低且适合双均线策略的标的，可能效果要更好一些。
> 三个组合，一个单吊创业板 ETF ，一个一大堆 ETF ，一个介于中间，看起来好像分散风险一样，但从整体持仓来看并没有分散，只是增加了一些鸡肋标的拉低收益而已。你可以算下整体波动，应该不会比单个组合小很多，整体波动不见得降低了整体收益却实打实降低了。
> 标的选择多了，看似可以到处开花分散风险降低波动，但也可能到处陷阱增加风险增加波动，这种做法是双刃剑，事实上由于核心之外的指数大多品质不好，所以陷阱的次数可能更多。
> 其实指数 ETF 本身包含的标的就很比较分散了，ETF 上面就没必要太过分散了，如果买一堆行业 ETF ，最后不就等于买了宽基，由于权重的原因，可能资金最终都是在那少数几十个大票上面。
> 而且同样的策略分成三个帐户去做，这个资金流动也不是很灵活啊，如果 2 号组合清仓了，而别的组合需要资金呢？这就不是市场造成的资金闲置成本了，而是资金管理导致的成本

> @bmpidev2019 @[grindsgears](https://v2ex.com/member/grindsgears) 那个 ppt 的确是梳理我看的一些书的知识结合自己的实践，但那是 16 年的事情了，里面很多方法其实我现在并没有采用，不过作为了解的一个输入还是很有必要的。至于自相矛盾那是一个问题，当你能把矛盾都识别清楚，那我觉得的确 get 到真正的知识了，目前我还在尝试寻找适合我的方式。市场上能赚钱的方式很多，找到适合自己的就 OK ，我现在的投资组合的问题在于我没有严格按照某个规则去执行，一直在尝试不同的策略，这也导致第三个组合没有前面两个组合那么好的收益，至于用未公开的知识赚钱这个我觉得只是一方面，公开的策略依旧可以盈利，因为大部分人是不用策略的，小赚总是可以的，这种方式我觉得对大多数人来说足够了，我并不期望能赚钱到专业玩家这种收益，对上班族也不现实。还有交易这东西很难说频率上去了盈利高就找到了圣杯，曾经 16 年到 17 年我当时在家失业有时间一直高频率交易，也能拿到很高的收益（半年 30%这种，跑赢大盘不少），但非常耗费时间，加上本金没那么多，这是一个非常浪费时间的行为，所以我觉得适合我的就是不需要关注的策略，能得到不算差的收益就可以了，因为时间还是要投入到自己专业技能上提高本金才是王道，当然我们讨论这么久，也说明不同人有不同适合他的交易方式，这是非常因人而异的事情。
> @[nguoidiqua](https://v2ex.com/member/nguoidiqua) 我那个交易信号有两组，第一组就是我回测双均线效果好的十个指数，后面都是其他的。过于分散是个问题，这也是未来在策略设计上需要多试验的，这个系统在设计时已经把策略和机器人接口设计好了，我们可以加多种策略对比，然后用一策略多种标的单一标的同时对比，但这样我担心过拟合的问题，不过可以尝试试验。指数基金虽然已经分散了，但还有相关系数低的指数，把相关系数低的组合起来是可以降低风险的，当然就看怎么配比了，这也是需要测试的。

> @nguoidiqua @[grindsgears](https://v2ex.com/member/grindsgears)
> 其实劝人是看人来的，你没有教楼主学你那么搞，因为你知道不适合楼主。但其实分散配置什么拿着不动什么的，也不适合楼主的。
> 楼主这么多年下来，肯定是有过各种尝试的，这几年稳定在这个方法上，显然这种搞法和他自身性格比较融洽，从数据来看目前也还可以，当然 18 年后面三年有点运气成分，不过长远来说应该比拿着不动稍好一些。
> 既然这个适合他，那在这个基础上做得更好就行了，换别的未必就会更好。

> @bmpidev2019 @[nguoidiqua](https://v2ex.com/member/nguoidiqua) 刚突然想到，很多时候我们讨论，忽略了投资组合金额的规模了，同一策略分散不分散与本金规模有关系，如果是千万级别的组合，那因为流动性与交易冲击的问题，肯定要多分散，哪怕降低收益也有必要，如果几十万的确没必要分散，如第二个组合一样集中交易一个也可以，但这里存在的问题是，第二个组合有运气成分，因为我们不知道某个指数到底未来收益行不行，如果运气不好，几年交易恒生指数，那收益一定很惨，虽然比单独拿着可能要好一些，所以分散也是有一定必要性的，就看怎么权衡了。从相关系数的角度去分配金额是个办法，但难点在于很难搞清楚当前的指数投资交易价值的高低，如果某个市场处于顶峰，这时候啥策略也别管，空仓不交易就是最好的，我曾经尝试用估值的办法去与均线策略结合，但难度太高了，微操到最后还不如单纯一个策略，这也是我驾驭不了不同策略的内在矛盾性，也许有人能搞的了吧，但难度不是一般的高。

> @nguoidiqua @[bmpidev2019](https://v2ex.com/member/bmpidev2019)
> 先把三个组合加总算算波动回撤然后和它们单独对比看看。
> 你目前这个数据，1 号的回撤比 3 号反而小，甚至 2 号也比 3 号小，1 号和 2 号和差别也不算很大，当然这个可能和资金量也有点关系。
> 这个策略本身就包含了止损这种控制回撤的操作，波动这方面真的不需要考虑太多。很多时候过度考虑不但起不到作用，反而增加了多余的成本。太多的决策因素反而会降低决策质量，这是走向成熟投资者需要突破的一个思想关隘，不要追求广而全。
> 找几个相关性低的标的，主要是为了提高资金利用率，降低风险是次要的。其实你这个不是资产配置，标的也不是同时都持有，未必能起到互相抵消波动的作用。
> 现在制约你收益率提高的瓶颈主要在两个方面吧，一个是一些资金用在了一些收益低的标的上，重要的标的仓位不够。另一个就是有些资金被不必要的闲置了，这是你组合管理造成的。
> 你的博客我看了有一年多了，年初跌的那阵你改了交易信号触发条件我也看到了，太怕回撤就会损失收益啊。

> @bmpidev2019 @[nguoidiqua](https://v2ex.com/member/nguoidiqua) 三号我微操了啊，我买了一些中概股 etf （让组合整体亏损了 10%左右）。。。所以我不能微操，这是一个教训，放大了亏损，我的错误在于我把中概用估值的方法去交易了，显然完全错误。当然我也微操了加密币，不过没放入这个投资组合，加密币的盈利弥补了中概的亏损😂
> 波动性方面我的确过于谨慎了，可以适当收缩投资组合的范围，集中到不超过五个相关系数低的适合双均线的投资标的来，这是未来的计划。
> 好处在于我现在投入的本金不多，主要是测试提升我的交易能力，目前陷入了一个瓶颈，你和 @[grindsgears](https://v2ex.com/member/grindsgears) 都提供了不错的反馈，谢谢你们！
> 资金利用率不高，我现在想到的办法是适当加入一些债券的配置，而非闲置等待交易信号。但还是资金比较少，我本身也懒得去操作，因为之前都需要手工记录交易，比较麻烦，现在这个系统可以自动做了，可以适当调整下。

> @nguoidiqua @bmpidev2019 资金量这个问题我是首先就会考虑的，交易量大的标的才会选择，虽然我资金量很小哈哈（习惯早养成）。
> 分散是个度的问题，就好像双均线选择长一点还是短一点也会有些区别，适当分散能增加资金利用率，是能提升收益率的，但随着数量的增加边际收益就越来越少了，到了某个临界点反而因为纳入一些较差的标的，开始降低收益率。
> 具体多少标的合适不同的投资大家都有不同看法，你可以先从减法做起，把一些重合的去掉，把一些很鸡肋的去掉，然后考虑那些领域没有覆盖到，有没有好的标的选择。其实大部分投资大家都是靠少数几个关键标的赚大钱的，其他的就是投石问路的石头。
> 
> 估值和均线结合，我是不看好的，其他多指标结合我也不是很看好。太多指标会导致更难判断，而且表现一致的时候比较少，出手少了虽然成功率高了但总体收益反而低了。而且，多指标表现一致的时候，也不一定就是对的。
> 比如市场顶峰就空仓不交易，这个做法未必比只看双均线好，顶峰也可以再大涨的。

> @grindsgears @[bmpidev2019](https://v2ex.com/member/bmpidev2019)
> @[nguoidiqua](https://v2ex.com/member/nguoidiqua)
> 1.相关性!=因果关系
> 2.bmpidev2019 提的建议非常的实用，这个系统跟踪这么多 etf ，目的是为了增加分散，降低风险，但是增加了流动性风险和减少了收益。而且，参考我下面所说，对于个人投机者，不是投资者， 分散根本就没有什么意义。
> 3.就中国股市而言，500etf 的 benchmark 应该是年化 10%左右。 但是绝对不是 evenly distributed ，也就意味这，你这种 trend following 在大部分时间都是亏钱的。必须要抓住 tail events 才可以。那么分散化去做就没有什么意义，不如平时拿点无风险收益。
> 4.你把系统弄的过于复杂了，而且没有找到真正你相信的 edge 。所以才会不停的微调，不出所料，以后你还会接着微调
> 5.投资组合金额的规模是很重要，但是，更重要的是目的，基金经理的目的可能不是稳定的收益而是打响名声然后收管理费。 分散对个人而言根本没有什么太大的意义。 你本金 10w ，翻倍了也就 20w ，回撤就更不重要了 lol 。
> 6.没有最好的，只有适合你的，但是大部分人做的事情，应该没有什么优势。比如我看有人弄什么网格交易，不停的 buy dips 会爆飞，你反过来 buy breakout 说不定还可以。因为真正 tail evnets 来的时候，你可以赚到过早卖出人的钱，前提是 breakout 后要 trailing 。
> 7, 有好多东西，是书本，教材里面相冲突的。不如前几年你 buy atm call + sell atm put ，这个组合可以杀到爆。
> 8,我个人觉得，etf 也许是一个好的投资工具，但是绝对不是好的投机工具。 还是那句话，翻倍也无法改变你的人生。不过 etf 里面有不少不为人知的机会，我就不多说了。
> 9, 我个人觉得，对于个人投机者来说，最爽的应该是，把大部分资金都放在无风险收益里面，一旦时间成熟，高 sigma 的时候，赌一把 option all in ，最好是 otm 那种，亏了也无所谓，要赚就赚起飞。一年做个几次，算起来不比你这种低。还没有负担。 不过还是要看个人的喜好。这种策略就是日常稳定亏，但是一旦遇到爽的时候，一次吧一年钱都赚了。心理上比较难接受，我有时候 all in etf 同时买 otm call 。😂 永不 hedge

> @grindsgears @nguoidiqua @[nguoidiqua](https://v2ex.com/member/nguoidiqua) 比如市场顶峰就空仓不交易，这个做法未必比只看双均线好，顶峰也可以再大涨的。
> 
> 老哥稳，，这是一个典型的 hindsight bias 。 在这种大家都跑路的时候，要勇于做 breakout,等大家反应过来的时候再卖给他们，同样，在下跌的时候要勇于买 put, sell rallies ，永远不要 buy dips... lool,

> @bmpidev2019 @[grindsgears](https://v2ex.com/member/grindsgears) 有意思的洞见。我曾经有段时间也想通过类似的方式去博一把，但实践下来就是累，虽然可以拿到不错的收益，但是本金太少，耗费的时间很多，总账算下来还不如把时间放在技能的提高上。假设有 100 万的本金，一年 20%的收益也不过 20 万，但技能的提升却比较容易的提高到超过 20 万甚至更高的层级上。我们的工作其实折现后就是一个恐怖的本金，年薪 50 万，以 10%的收益折算就是 500 万的本金的投资组合，但真正要搞 500 万的本金还是挺难的。不过这是另外一个关于衡量时间投入在投资还是技能上的问题。
> 回到投资的问题上来。ETF 是个不错的适合懒人的投资工具，对我来说，能超越 10%-15%的 CAGR 回报我就满意了，因为我的时间需要投入到本金的提高上来。当然未来年纪大了，本金多了，时间可以更多投入的投资上，但那时候我的风险承受能力会进一步降低，这些因素决定了我不能玩我玩不了的策略（如果抛开这些因素，我当然可以玩飞这些策略，大不了也就是产生一定的亏损）。

> @bmpidev2019 @[nguoidiqua](https://v2ex.com/member/nguoidiqua) 策略简单些更好适应市场的变化，复杂的策略其实就是在过拟合，回测也很容易犯这个错误。我特意选择粗糙的策略，及分散的标的都是在回避过拟合的可能性。

**Comment by Microndgt** [comment]
哈哈 博主跟我思路差不多 我也是搞了一套策略 写了一套交易系统 用来发信号 回测 目前收益率也还行 今年25%

**Comment by baiwei666** [comment]
老哥，你这个系统怎么使用啊，摸不着头脑😂

**Reply by madawei2699** [reply]
直接订阅邮件就行了，也可以在网页看这些投资组合的表现啊，我不理解你的问题在哪里？

**Comment by shellsec** [comment]
话说，我可以自建吗？有搭建文档吗？
